spec/controllers/users_controller_spec.rb

```diff
@@ -67,13 +67,15 @@
       it 'gets her membership in the current organization' do
         get :index
 
-        expect(assigns(:memberships)).to eq({
-          member.user_id => member,
-          another_member.user_id => another_member,
-          member_admin.user_id => member_admin,
-          wrong_email_member.user_id => wrong_email_member,
-          empty_email_member.user_id => empty_email_member
-        })
+        assigns(:members).each { |m| expect(m).to be_a(MemberDecorator) }
```

Comment:
IMO it's a not a good idea to couple to particular class types. Makes future changes harder and it's sort of against Ruby's duck-type principle.

Reply:
Ok but still somewhat I want to specify that what's sent to the view is a list of decorated objects. What about a general method `decorated_object` on the decorator? Instead of `member`. I have used `member` just for laziness, for instance when using draper I tend to use `object` instead of the named accessor.